### ISSUE TITLE
Advanced Generators update crash

### ISSUE THREAD

**Issue by Jemsterr** [yuval9313/ONI-Mods#4]
[Player.log](https://github.com/yuval9313/ONI-Mods/files/6270994/Player.log)

Here's the crashlog.  I tried unsubscribing, resubscribing, or here just deleting the folder and letting ONI readd it.  In every case, ONI blames the crash on other mods, but unsubscribing from this mod fixes the issue.

Hopefully there's something in here that helps you locate the crash.

**Comment by Jemsterr** [comment]
Still crashing after the update on 8 April.  I have deleted the folder and let Steam re-download it.  I have removed all other mods that ONI complained about during the crash, but still get the crash.  Here is my player.log
[Player.log](https://github.com/yuval9313/ONI-Mods/files/6277079/Player.log)

**Comment by yuval9313** [comment]
I'm not sure if this is really the latest version, I've added a line which tells that this is version 1.1.0 in the log, this is how I can control the versions and debug it correctly, I've searched the log for this line and I couldn't find it, I think you should try Mod Updater or unsub and resub to the MOD

**Comment by Jemsterr** [comment]
It seems it wasn't.  I don't know how Steam handles its mod versioning, but either I kept getting a cached old version or I don't know what. 
Finally, the next day, I resubscribed and it gave me the latest version (with the archived folder) and the game loads.
I don't know if it was loading ONI and exiting a couple times without it subscribed, or just the fact that enough time had passed, but now I have the latest version.  It's all running now.  I'll put a comment to that effect in Steam.
Thanks for your efforts.  Hope you get someone with the DLC get in contact with you.
